### PR TITLE
use both header and a query string parameter for the x-token-auth

### PR DESCRIPTION
**Pull request by alexginzburg**
addresses https://refinery29.atlassian.net/browse/HD-2953

- checks query string parameter in addition to the header
- use either value
- use the value as is, no need chop the first 7 characters

### CODE REVIEW

File: lib/multi-auth.lua [hunk 1]
```diff
@@ -8,13 +8,14 @@ local whitelist_ips = multiauth_ip_whitelist or {}
 
 whitelist = iputils.parse_cidrs(whitelist_ips)
 
-local token_auth = ngx.var.http_x_token_auth
```
Comment (joeles) [inline lib/multi-auth.lua]: Couldn't this be something like this?
```lua
local token_auth = ngx.var.arg_x_token_auth or ngx.var.http_x_token_auth
```
Then we wouldn't have to change anything else.

Comment (alexginzburg) [reply lib/multi-auth.lua]: we should be able to. I'm not thinking straight today

**Comment by alexginzburg** [comment]
@joeles made a change, this looks much better. will test on stage

**Comment by alexginzburg** [comment]
@joeles pushed another commit to remove sub(token, 7)

**Pull request merged**